### PR TITLE
Kips code cleanup

### PR DESCRIPTION
**Pull request by kippetje2000**
Have a look if you like it

### CODE REVIEW

File: Defs/ResurrectionRitual/ResurrectionRitual_Ritual.xml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+		<defName>Resurrection</defName>
+		<label>resurrection</label>
+		<extraTextPawnDeathLetter>{PAWN_nameDef} should be brought back to life with dark, nearly blasphemous arts.</extraTextPawnDeathLetter>
+		<description>Tapping into untold necromantic powers, you revive a colonist's life</description>
+		<issue>Ritual</issue>
+		<impact>Medium</impact>
+		<iconPath>UI/Icons/Rituals/RitualFuneral</iconPath>
+		<preceptClass>Precept_Ritual</preceptClass>
+		<ritualPatternBase>ResurrectionRitual</ritualPatternBase>
+		<allowDuplicates>true</allowDuplicates>
```
Comment (arvalaan) [inline Defs/ResurrectionRitual/ResurrectionRitual_Ritual.xml:13]: <allowDuplicates> should be false.

File: Defs/ResurrectionRitual/ResurrectionRitual_Ritual.xml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+		<displayOrderInImpact>100</displayOrderInImpact>
+		<ignoreLimitsInEditMode>false</ignoreLimitsInEditMode>
+		<receivesExpectationsQualityOffset>true</receivesExpectationsQualityOffset>
+	</PreceptDef>
+	<RitualPatternDef>
+		<defName>ResurrectionRitual</defName>
+		<nameMaker>NamerRitualFuneral</nameMaker>
+		<ritualExpectedDesc>Once a {IDEO} member dies, other {MEMBERS} gather around their grave for {RITUAL_labelDef}.</ritualExpectedDesc>
+		<ritualExpectedPostfix>{0} {1} want to carry out the funeral as soon as possible.</ritualExpectedPostfix>
+		<ritualObligationTriggers>
```
Comment (arvalaan) [inline Defs/ResurrectionRitual/ResurrectionRitual_Ritual.xml:25]: I feel like there shouldn't be an obligationTrigger for resurrection. It should be a choice, not an expectation

**Review by arvalaan — CHANGES_REQUESTED** [review]

File: Defs/ResurrectionRitual/ResurrectionRitual_Ritual.xml (hunk 1 above)
Comment (kippetje2000) [reply Defs/ResurrectionRitual/ResurrectionRitual_Ritual.xml:13]: I did and it keeps giving errors so im not sure where it went wrong

File: Defs/ResurrectionRitual/ResurrectionRitual_Ritual.xml (hunk 2 above)
Comment (kippetje2000) [reply Defs/ResurrectionRitual/ResurrectionRitual_Ritual.xml:25]: Is this that message on the right of the screen?

**Comment by arvalaan** [comment]
I'll merge your PR and look into it :)

**Pull request merged**